`make` fails due to doctests

nosetests -s --with-doctest --doctest-tests --doctest-extension=rst \
		--doctest-extension=inc --doctest-fixtures=_fixture doc/ \
	
	nose.plugins.cover: ERROR: Coverage not available: unable to import coverage module
	Failure: ImportError (No module named sphinxgallery) ... ERROR
	Doctest: introduction.rst ... FAIL
	
	======================================================================
	ERROR: Failure: ImportError (No module named sphinxgallery)
	----------------------------------------------------------------------
	Traceback (most recent call last):
	  File "/Users/filo/anaconda/lib/python2.7/site-packages/nose/loader.py", line 418, in loadTestsFromName
	    addr.filename, addr.module)
	  File "/Users/filo/anaconda/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
	    return self.importFromDir(dir_path, fqname)
	  File "/Users/filo/anaconda/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
	    mod = load_module(part_fqname, fh, filename, desc)
	  File "/Users/filo/krzysztof.gorgolewski@gmail.com/workspace/nilearn/doc/conf.py", line 24, in <module>
	    import sphinxgallery
	ImportError: No module named sphinxgallery
	
	======================================================================
	FAIL: Doctest: introduction.rst
	----------------------------------------------------------------------
	Traceback (most recent call last):
	  File "/Users/filo/anaconda/lib/python2.7/doctest.py", line 2226, in runTest
	    raise self.failureException(self.format_failure(new.getvalue()))
	AssertionError: Failed doctest test for introduction.rst
	  File "/Users/filo/krzysztof.gorgolewski@gmail.com/workspace/nilearn/doc/introduction.rst", line 0
	
	----------------------------------------------------------------------
	File "/Users/filo/krzysztof.gorgolewski@gmail.com/workspace/nilearn/doc/introduction.rst", line 274, in introduction.rst
	Failed example:
	    smoothed_img = image.smooth_img("/home/user/t_map000.nii", fwhm=5)
	Exception raised:
	    Traceback (most recent call last):
	      File "/Users/filo/anaconda/lib/python2.7/doctest.py", line 1315, in __run
	        compileflags, 1) in test.globs
	      File "<doctest introduction.rst[1]>", line 1, in <module>
	        smoothed_img = image.smooth_img("/home/user/t_map000.nii", fwhm=5)
	      File "/Users/filo/krzysztof.gorgolewski@gmail.com/workspace/nilearn/nilearn/image/image.py", line 253, in smooth_img
	        img = check_niimg(img)
	      File "/Users/filo/krzysztof.gorgolewski@gmail.com/workspace/nilearn/nilearn/_utils/niimg_conversions.py", line 157, in check_niimg
	        niimg = load_niimg(niimg)
	      File "/Users/filo/krzysztof.gorgolewski@gmail.com/workspace/nilearn/nilearn/_utils/niimg.py", line 50, in load_niimg
	        niimg = nibabel.load(niimg)
	      File "/Users/filo/anaconda/lib/python2.7/site-packages/nibabel/loadsave.py", line 44, in load
	        return guessed_image_type(filename).from_filename(filename, **kwargs)
	      File "/Users/filo/anaconda/lib/python2.7/site-packages/nibabel/loadsave.py", line 76, in guessed_image_type
	        with BinOpener(filename) as fobj:
	      File "/Users/filo/anaconda/lib/python2.7/site-packages/nibabel/openers.py", line 74, in __init__
	        self.fobj = opener(fileish, *args, **kwargs)
	    IOError: [Errno 2] No such file or directory: '/home/user/t_map000.nii'
	----------------------------------------------------------------------
	File "/Users/filo/krzysztof.gorgolewski@gmail.com/workspace/nilearn/doc/introduction.rst", line 281, in introduction.rst
	Failed example:
	    smoothed_img.to_filename("/home/user/t_map000_smoothed.nii")
	Exception raised:
	    Traceback (most recent call last):
	      File "/Users/filo/anaconda/lib/python2.7/doctest.py", line 1315, in __run
	        compileflags, 1) in test.globs
	      File "<doctest introduction.rst[2]>", line 1, in <module>
	        smoothed_img.to_filename("/home/user/t_map000_smoothed.nii")
	    NameError: name 'smoothed_img' is not defined
	----------------------------------------------------------------------
	File "/Users/filo/krzysztof.gorgolewski@gmail.com/workspace/nilearn/doc/introduction.rst", line 289, in introduction.rst
	Failed example:
	    first_volume = image.index_img("/home/user/fmri_volumes.nii", 0)
	Exception raised:
	    Traceback (most recent call last):
	      File "/Users/filo/anaconda/lib/python2.7/doctest.py", line 1315, in __run
	        compileflags, 1) in test.globs
	      File "<doctest introduction.rst[3]>", line 1, in <module>
	        first_volume = image.index_img("/home/user/fmri_volumes.nii", 0)
	      File "/Users/filo/krzysztof.gorgolewski@gmail.com/workspace/nilearn/nilearn/image/image.py", line 521, in index_img
	        imgs = check_niimg_4d(imgs)
	      File "/Users/filo/krzysztof.gorgolewski@gmail.com/workspace/nilearn/nilearn/_utils/niimg_conversions.py", line 240, in check_niimg_4d
	        return check_niimg(niimg, ensure_ndim=4, return_iterator=return_iterator)
	      File "/Users/filo/krzysztof.gorgolewski@gmail.com/workspace/nilearn/nilearn/_utils/niimg_conversions.py", line 157, in check_niimg
	        niimg = load_niimg(niimg)
	      File "/Users/filo/krzysztof.gorgolewski@gmail.com/workspace/nilearn/nilearn/_utils/niimg.py", line 50, in load_niimg
	        niimg = nibabel.load(niimg)
	      File "/Users/filo/anaconda/lib/python2.7/site-packages/nibabel/loadsave.py", line 44, in load
	        return guessed_image_type(filename).from_filename(filename, **kwargs)
	      File "/Users/filo/anaconda/lib/python2.7/site-packages/nibabel/loadsave.py", line 76, in guessed_image_type
	        with BinOpener(filename) as fobj:
	      File "/Users/filo/anaconda/lib/python2.7/site-packages/nibabel/openers.py", line 74, in __init__
	        self.fobj = opener(fileish, *args, **kwargs)
	    IOError: [Errno 2] No such file or directory: '/home/user/fmri_volumes.nii'
	----------------------------------------------------------------------
	File "/Users/filo/krzysztof.gorgolewski@gmail.com/workspace/nilearn/doc/introduction.rst", line 293, in introduction.rst
	Failed example:
	    for volume in image.iter_img("/home/user/fmri_volumes.nii"):
	        smoothed_img = image.smooth_img(volume, fwhm=5)
	Exception raised:
	    Traceback (most recent call last):
	      File "/Users/filo/anaconda/lib/python2.7/doctest.py", line 1315, in __run
	        compileflags, 1) in test.globs
	      File "<doctest introduction.rst[4]>", line 1, in <module>
	        for volume in image.iter_img("/home/user/fmri_volumes.nii"):
	      File "/Users/filo/krzysztof.gorgolewski@gmail.com/workspace/nilearn/nilearn/image/image.py", line 537, in iter_img
	        return check_niimg_4d(imgs, return_iterator=True)
	      File "/Users/filo/krzysztof.gorgolewski@gmail.com/workspace/nilearn/nilearn/_utils/niimg_conversions.py", line 240, in check_niimg_4d
	        return check_niimg(niimg, ensure_ndim=4, return_iterator=return_iterator)
	      File "/Users/filo/krzysztof.gorgolewski@gmail.com/workspace/nilearn/nilearn/_utils/niimg_conversions.py", line 157, in check_niimg
	        niimg = load_niimg(niimg)
	      File "/Users/filo/krzysztof.gorgolewski@gmail.com/workspace/nilearn/nilearn/_utils/niimg.py", line 50, in load_niimg
	        niimg = nibabel.load(niimg)
	      File "/Users/filo/anaconda/lib/python2.7/site-packages/nibabel/loadsave.py", line 44, in load
	        return guessed_image_type(filename).from_filename(filename, **kwargs)
	      File "/Users/filo/anaconda/lib/python2.7/site-packages/nibabel/loadsave.py", line 76, in guessed_image_type
	        with BinOpener(filename) as fobj:
	      File "/Users/filo/anaconda/lib/python2.7/site-packages/nibabel/openers.py", line 74, in __init__
	        self.fobj = opener(fileish, *args, **kwargs)
	    IOError: [Errno 2] No such file or directory: '/home/user/fmri_volumes.nii'

--- a/doc/introduction.rst
+++ b/doc/introduction.rst
@@ -287,7 +287,7 @@ first have to import the :ref:`plotting <plotting>` functionality by::
 Then you can call the function that creates a "glass brain" by giving it
 the file name::
 
-    >>> plotting.plot_glass_brain("/home/user/t_map000.nii")
+    >>> plotting.plot_glass_brain("/home/user/t_map000.nii")   # doctest: +SKIP
 
 .. image:: auto_examples/manipulating_visualizing/images/plot_demo_glass_brain_001.png
     :target: auto_examples/manipulating_visualizing/plot_demo_glass_brain.html
@@ -308,14 +308,14 @@ the :mod:`nilearn.image` module for image manipulation, e.g.
 :func:`image.smooth_img` for smoothing::
 
     >>> from nilearn import image
-    >>> smoothed_img = image.smooth_img("/home/user/t_map000.nii", fwhm=5)
+    >>> smoothed_img = image.smooth_img("/home/user/t_map000.nii", fwhm=5)   # doctest: +SKIP
 
 The returned value `smoothed_img` is a `NiftiImage object
 <http://nipy.org/nibabel/nibabel_images.html>`_. It can either be passed
 to other nilearn functions operating on niimgs (neuroimaging images) or
 saved to disk with::
 
-    >>> smoothed_img.to_filename("/home/user/t_map000_smoothed.nii")
+    >>> smoothed_img.to_filename("/home/user/t_map000_smoothed.nii")   # doctest: +SKIP
 
 Finally, nilearn deals with Nifti images that come in two flavors: 3D
 images, which represent a brain volume, and 4D images, which represent a
@@ -323,11 +323,11 @@ series of brain volumes. To extract the n-th 3D image from a 4D image, you can
 use the :func:`image.index_img` function (keep in mind that array indexing
 always starts at 0 in the Python language)::
 
-    >>> first_volume = image.index_img("/home/user/fmri_volumes.nii", 0)
+    >>> first_volume = image.index_img("/home/user/fmri_volumes.nii", 0)   # doctest: +SKIP
 
 To loop over each individual volume of a 4D image, use :func:`image.iter_img`::
 
-   >>> for volume in image.iter_img("/home/user/fmri_volumes.nii"):
+   >>> for volume in image.iter_img("/home/user/fmri_volumes.nii"):   # doctest: +SKIP
    ...     smoothed_img = image.smooth_img(volume, fwhm=5)
 
 .. topic:: **Exercise: varying the amount of smoothing**

--- a/setup.cfg
+++ b/setup.cfg
@@ -12,7 +12,7 @@ cover-package = nilearn
 #pdb-failures = 1
 with-doctest=1
 doctest-extension=rst
-ignore-files=plot_.*.py
+ignore-files=(plot_.*.py|conf\.py)
 
 [wheel]
 universal=1

